src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/types/openapi.d.ts

```diff
@@ -1,517 +1,636 @@
-import type { AxiosRequestConfig, OpenAPIClient, OperationResponse, Parameters, UnknownParamsObject } from "openapi-client-axios";
```

Comment:
We should add this as a git hook but can you please format everything (it makes the openapi.d.ts changes more readable in PR):

`npx prettier --write .`

If you're using VScode you should be able to turn on auto formatting in `settings.json` moving forward. For now we can just add `"format": "prettier --write .",` to `package.json` scripts and run `npm run format`